patterns/mp4.hexpat

```diff
@@ -199,7 +208,6 @@ struct SubMediaBox {
         ("mdhd"): MediaHeaderBox box [[inline]];
         ("hdlr"): HandlerBox box [[inline]];
         ("minf"): MediaInformationBox box [[inline]];
-        ("sbtl"): SampleBoxTable box [[inline]];
```

Comment:
I did this in my previous PR,  which was wrong, and why it didnt find any childboxes before
Previous PR: https://github.com/WerWolv/ImHex-Patterns/pull/185/files